_overviews/scala3-book/taste-repl.md

```diff
@@ -11,46 +11,74 @@ next-page: taste-vars-data-types
 The Scala REPL (“Read-Evaluate-Print-Loop”) is a command-line interpreter that you use as a “playground” area to test your Scala code.
 You start a REPL session by running the `scala` or `scala3` command depending on your installation at your operating system command line, where you’ll see a “welcome” prompt like this:
 
+
+{% tabs command-line class=tabs-scala-version %}
+
+{% tab 'Scala 3' for=command-line %}
```

Comment:
I hadn't noticed that the scala 3 tab came first until I saw your comment, for consistency please would you swap the tabs? :) - We will also likely rebrand the book to just the Scala Book when we have updated enough code examples